tasks/main.yml

```diff
@@ -2,10 +2,10 @@
 - block:
   - name: Login to oVirt
     ovirt_auth:
-      url: "{{ engine_url | default(omit) }}"
-      username: "{{ engine_user | default(omit) }}"
-      password: "{{ engine_password | default(omit) }}"
-      ca_file: "{{ engine_cafile | default(omit) }}"
+      url: "{{ engine_url | default(lookup('env','host')) | default(omit) }}"
```

Comment:
I think it's called `OVIRT_URL`, and same for other env variables. Did you test it? Does it work?